Additional os specific installation instruction

ProphetLamb opened the pull request:
Documented additional installation instruction required for Windows and Linux, respectively.

ProphetLamb committed:
Installation under Windows

ProphetLamb committed:
Installation under Linux

ProphetLamb committed:
Remove su requirement

Siphalor commented on index.md:
````diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
+Now for the actual installation execute the following commands (tinytex ist a texlive distribution)
+```pwsh 
+scoop bucket add r-bucket https://github.com/cderv/r-bucket.git
+scoop install tinytex pandoc pandoc-crossref
````
Not sure if tinytex make stuff simpler or more advanced - `miktex` would probably be the easy solution.

Siphalor commented on index.md:
````diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
+test -d /home/linuxbrew/.linuxbrew && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+test -r ~/.bash_profile && echo "eval \"\$($(brew --prefix)/bin/brew shellenv)\"" >> ~/.bash_profile
+echo "eval \"\$($(brew --prefix)/bin/brew shellenv)\"" >> ~/.profile
+```
````
That wouldn't even install Homebrew.
I think what I'm looking at is an overly complicated way to add the Homebrew environment variables to the profile.

The [official website](https://brew.sh) uses a way easier installation command, that should do all the heavy lifting:

```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Siphalor commented on index.md:
````diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
+```bash
+python -m ensure pip
+brew install pandoc-crossref texlive
+pip install pandoc pandoc-acro pandoc-include--user
````
I think that `--user` bit there needs an additional space.

Siphalor commented on index.md:
```diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
 
+### Windows
+
+Start powershell
```
Please elaborate these statements to a full sentence.
Also, PowerShell is only required for the Scoop installation, not for any further steps - so it might make sense moving it into that section.

Siphalor commented on index.md:
```diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
+
+### Linux
+
+Start terminal
```
That's probably obvious to a Linux user, anyway move this down to the section below.

Siphalor commented on index.md:
```diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
+
+Start powershell
+
+**If scoop is not installed**
```
Some colons would be nice here, to indicate that these are meant as section heading.

Siphalor commented on index.md:
````diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
+scoop install uutils-coreutils
+```
+
+Now for the actual installation execute the following commands (tinytex ist a texlive distribution)
````
Since the above sections use full code blocks, it is hard to distinguish between the "GNU Make" section and this one.

These last general sections should have their own headings, such as "Eigentliche Installation"

Siphalor reviewed (changes requested):
While I understand the general idea, this pr has some way left to go:

- First off, this project's content is written in German and should remain German.
- All proper names should use the appropriate capitalization (at least the first letter).
- Fix all of the comments :3

Siphalor commented:
Also, I just implemented enabled the CI for pull requests - so please merge from `main` :)

ProphetLamb committed:
Merge branch 'Siphalor:main' into main

ProphetLamb committed:
German prosa
- install pandoc via brew because pandoc-crossref depends on pandoc
- mention MikTex alternative for windows.

ProphetLamb committed:
Links instead of inline code

Siphalor committed:
Grammar, wording, spelling, capitalization etc.

Siphalor reviewed (approved):
Fixed up the rest of the spelling, grammar, wording and other issues :)
